### PR TITLE
1.4.0 release candidate

### PR DESCRIPTION
**Pull request by stevenweaver**

### CODE REVIEW

**Comment by stevenweaver** [comment]
@stephenshank - a number of updates here. I'm not sure who `bitcore` is. Would you consider the parts you're aware of ready for review?

**Comment by stephenshank** [comment]
@stevenweaver Yes, this looks ready for review. I am not sure who @bitcore is either.

I have a few changes I'd like to include in the next release and will push here/open up PRs in respective repos, if that's alright. Also I'll set this up on staging for testing.

**Comment by stevenweaver** [comment]
Awesome. Yes, it can just be pushed to develop then at this point since it will be included in the next release. This release includes the long-awaited configurable processors per analysis.

**Comment by spond** [comment]
@bitcore was the name of the group that Jason and I ran for UCSD CFAR;) Long dead.

**Comment by bitcore** [comment]
@stevenweaver Please check your email (.org) when you have a moment

**Comment by stevenweaver** [comment]
> @bitcore was the name of the group that Jason and I ran for UCSD CFAR;) Long dead.

Holds a special place in my heart. RIP @bitcore.

File: Makefile [hunk 1]
```diff
@@ -5,10 +5,8 @@ all: install
 hyphy:
 	echo "installing hyphy"
 	@if ! test -d ./.hyphy; then git clone http://github.com/veg/hyphy.git ./.hyphy/; fi
-	@cd ./.hyphy && git checkout master && git pull && git checkout 2.3.14 && cmake . && make -j 4 HYPHYMP && make -j 4 HYPHYMPI && cd ../
-	@if ! test -d ./.hyphy_gard_version2_3_11; then git clone http://github.com/veg/hyphy.git ./.hyphy_gard_version2_3_11/; fi
-	@cd ./.hyphy_gard_version2_3_11 && git checkout master && git pull && git checkout 2.3.11 && cmake . && make -j 4 HYPHYMP && make -j 4 HYPHYMPI && cd ../
-
+	@cd ./.hyphy && git checkout 2.5.0 && cmake . && make -j 4 hyphy && make -j 4 HYPHYMPI && cd ../
```
Comment (stephenshank) [inline Makefile]: I think we should consider bumping HyPhy to 2.5.1 and adding the `-DNOAVX=ON` to cmake.

**Pull request merged**